src/lattice/lattice_bravais.ts

```diff
@@ -1,26 +1,62 @@
 import constants from "../constants";
 import math from "../math";
-import { LATTICE_TYPE_CONFIGS } from "./types";
+import { LATTICE_TYPE_CONFIGS, LatticeType, Vector, VectorsAsArray } from "./types";
+
+export interface BravaisConfig {
```

Comment:
We should be able to get this and the below from ESSE schema, no?

Reply:
Right. Replaced this with `LatticeImplicitSchema`